### ISSUE TITLE
`no-unresolved` not working

### ISSUE THREAD

**Issue by vicasas** [import-js/eslint-plugin-import#2321]
I am experiencing the following issues in these two scenarios:

1. In versions equal to or less than 2.24.x:

The `no-unresolved` rule stopped working for some unknown reason.

When I import a module that does not exist the only error it gives me is the `no-unused-vars` but it does not tell me that the import does not exist (`no-unresolved`)

2. In versions equal to or greater than 2.25.x

The `no-unresolved` rule stopped working for some unknown reason, but another rule called `named` was triggered indicating that the module does not exist in that file.

Why does this happen? Is 1 and 2 expected to happen?

<details>
<summary>My settings:</summary>

```json
"devDependencies": {
    "eslint": "^7.32.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-config-next": "12.0.6",
    "eslint-config-prettier": "^8.3.0",
    "eslint-plugin-import": "^2.24.2",
    "eslint-plugin-jsx-a11y": "^6.4.1",
    "eslint-plugin-prettier": "^4.0.0",
    "eslint-plugin-react": "^7.25.1",
    "eslint-plugin-react-hooks": "^4.2.0",
    "husky": "^7.0.0",
    "lint-staged": "^12.1.2",
    "prettier": "^2.4.0"
  },
```

```json
{
  "env": {
    "browser": true,
    "es2021": true,
    "node": true,
    "jest": true
  },
  "extends": ["plugin:react/recommended", "plugin:@next/next/recommended", "airbnb", "prettier"],
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    },
    "ecmaVersion": 12,
    "sourceType": "module"
  },
  "plugins": ["react", "prettier"],
  "rules": {
    "prettier/prettier": "error",
    "react/react-in-jsx-scope": "off",
    "react/jsx-props-no-spreading": "off",
    "react/forbid-prop-types": "off",
    "react/require-default-props": [
      "error",
      {
        "ignoreFunctionalComponents": true
      }
    ]
  }
}
```
</details>

**Comment by soryy708** [comment]
Any noteworthy logs if you set this environment variable?

```
DEBUG=eslint-plugin-import:*
```
